index.html

```diff
@@ -1,91 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel ="stylesheet" href ="Currency.html"/>
-    <script src="Currency.JS"></script>
-    <title>Currency Converter App</title>
+    <title>Currency Conversion</title>
 </head>
-<body>
-    
-	<form id="currency_form">
-        <!-- Currency Converter -->
-        <h1 class="heading text-center display-2">
-            Currency Converter
-        </h1>
-        <hr>
-        
-     <div class="container">
-            <div class="main">
-                <div class="form-group">
-                    <label for="amount">
-                        Amount to Convert :
-                    </label>
-                    <input type="text" class="form-control searchBox" placeholder="0.00" id="oamount">
-                </div>
-                <div class="row">
-                    <div class="col-sm-6">
-                        <div class="input-group mb-3">
-                            <div class="input-group-prepend">
-                                <span class="input-group-text">From</span>
-                            </div>
-                        
-                            <select id="from-currency-dropdown">
-							<option value="">Select One …</option>
-							</select>
-							<label>Currency Code:</label>
+<body onload="populateCurrencyDropdown()">
```

Comment:
Prefer using `addEventListener` in the javascript file instead of HTML attributes.